docs/source/en/glossary.mdx

```diff
@@ -410,6 +451,16 @@ sequence, corresponding to the "question", has all its tokens represented by a `
 
 Some models, like [`XLNetModel`] use an additional token represented by a `2`.
 
+### transfer learning
+
+A technique which involves taking a model pretrained for a specific task and using it for something completely different. Instead of training a model from scratch, we can leverage knowledge obtained from an existing model as a starting point, which can speed up the learning process and reduce the amount of training data needed.
```

Comment:
```suggestion
A technique that involves taking a pretrained model and adapting it to a dataset specific to your task. Instead of training a model from scratch, you can leverage knowledge obtained from an existing model as a starting point. This speeds up the learning process and reduces the amount of training data needed.
```